<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskAnalyzerWindow</class>
 <widget class="QDialog" name="DiskAnalyzerWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>1200</width><height>750</height></rect>
  </property>
  <property name="windowTitle">
   <string>Disk Analyzer</string>
  </property>
  <property name="minimumSize">
   <size><width>1000</width><height>600</height></size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <property name="spacing"><number>4</number></property>
   <property name="leftMargin"><number>4</number></property>
   <property name="topMargin"><number>4</number></property>
   <property name="rightMargin"><number>4</number></property>
   <property name="bottomMargin"><number>4</number></property>
   
   <!-- MAIN CONTENT -->
   <item>
    <layout class="QHBoxLayout" name="mainContent">
     <property name="spacing"><number>6</number></property>
     
     <!-- LEFT: Disk Visualizations -->
     <item>
      <layout class="QVBoxLayout" name="diskLayout">
       
       <!-- DISK VIEWS (Side 0 + Side 1) -->
       <item>
        <layout class="QHBoxLayout" name="diskViewsLayout">
         <property name="spacing"><number>8</number></property>
         
         <!-- SIDE 0 -->
         <item>
          <widget class="QGroupBox" name="groupSide0">
           <property name="title"><string>Side 0</string></property>
           <property name="styleSheet"><string>QGroupBox { font-weight: bold; }</string></property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="labelSide0Info">
              <property name="text"><string>0 Tracks, 0 Sectors, 0 Bytes</string></property>
              <property name="styleSheet"><string>color: #00AA00; font-family: monospace;</string></property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSide0Format">
              <property name="text"><string>Format: Unknown</string></property>
              <property name="styleSheet"><string>font-family: monospace;</string></property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frameDisk0">
              <property name="minimumSize">
               <size><width>350</width><height>350</height></size>
              </property>
              <property name="frameShape"><enum>QFrame::Box</enum></property>
              <property name="styleSheet"><string>background-color: #1a1a1a;</string></property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         
         <!-- SIDE 1 -->
         <item>
          <widget class="QGroupBox" name="groupSide1">
           <property name="title"><string>Side 1</string></property>
           <property name="styleSheet"><string>QGroupBox { font-weight: bold; }</string></property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="labelSide1Info">
              <property name="text"><string>0 Tracks, 0 Sectors, 0 Bytes</string></property>
              <property name="styleSheet"><string>color: #00AA00; font-family: monospace;</string></property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSide1Format">
              <property name="text"><string>Format: Unknown</string></property>
              <property name="styleSheet"><string>font-family: monospace;</string></property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frameDisk1">
              <property name="minimumSize">
               <size><width>350</width><height>350</height></size>
              </property>
              <property name="frameShape"><enum>QFrame::Box</enum></property>
              <property name="styleSheet"><string>background-color: #1a1a1a;</string></property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         
        </layout>
       </item>
       
       <!-- VIEW CONTROLS -->
       <item>
        <widget class="QGroupBox" name="groupViewControls">
         <property name="title"><string>View</string></property>
         <property name="maximumHeight"><number>80</number></property>
         <layout class="QHBoxLayout">
          <item><widget class="QLabel"><property name="text"><string>Offset:</string></property></widget></item>
          <item>
           <widget class="QSlider" name="sliderOffset">
            <property name="orientation"><enum>Qt::Horizontal</enum></property>
            <property name="maximum"><number>100</number></property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinOffset">
            <property name="suffix"><string> %</string></property>
            <property name="maximum"><double>100.0</double></property>
            <property name="value"><double>85.0</double></property>
           </widget>
          </item>
          <item><widget class="QLabel"><property name="text"><string>X Scale:</string></property></widget></item>
          <item>
           <widget class="QSlider" name="sliderXScale">
            <property name="orientation"><enum>Qt::Horizontal</enum></property>
            <property name="maximum"><number>100</number></property>
            <property name="value"><number>50</number></property>
           </widget>
          </item>
          <item><widget class="QLabel"><property name="text"><string>Y Scale:</string></property></widget></item>
          <item>
           <widget class="QSlider" name="sliderYScale">
            <property name="orientation"><enum>Qt::Horizontal</enum></property>
            <property name="maximum"><number>100</number></property>
            <property name="value"><number>16</number></property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioTrackView">
            <property name="text"><string>Track view</string></property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioDiskView">
            <property name="text"><string>Disk view</string></property>
            <property name="checked"><bool>true</bool></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       
      </layout>
     </item>
     
     <!-- RIGHT: Status + Controls -->
     <item>
      <layout class="QVBoxLayout" name="rightPanel">
       <property name="spacing"><number>6</number></property>
       
       <!-- STATUS INFO -->
       <item>
        <widget class="QGroupBox" name="groupStatus">
         <property name="title"><string>Status</string></property>
         <property name="styleSheet"><string>QGroupBox { font-weight: bold; }</string></property>
         <property name="minimumWidth"><number>280</number></property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="labelTrackInfo">
            <property name="text"><string>Track: - Side: -</string></property>
            <property name="styleSheet"><string>font-family: monospace; font-weight: bold;</string></property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textSectorInfo">
            <property name="readOnly"><bool>true</bool></property>
            <property name="maximumHeight"><number>120</number></property>
            <property name="styleSheet"><string>font-family: monospace; font-size: 9pt;</string></property>
            <property name="plainText"><string>MFM Sector
Sector ID: -
Track ID: - Side ID: -
Size: - (ID: -)
Data checksum: -
Head CRC: -
Data CRC: -</string></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       
       <!-- HEX DUMP -->
       <item>
        <widget class="QGroupBox" name="groupHexDump">
         <property name="title"><string>Hex Dump</string></property>
         <property name="styleSheet"><string>QGroupBox { font-weight: bold; }</string></property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QTextEdit" name="textHexDump">
            <property name="readOnly"><bool>true</bool></property>
            <property name="styleSheet"><string>font-family: monospace; font-size: 8pt; background-color: #1a1a1a; color: #00ff00;</string></property>
            <property name="plainText"><string>00000  00 00 00 00 00 00 00 00  ........
00008  00 00 00 00 00 00 00 00  ........
00010  00 00 00 00 00 00 00 00  ........</string></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       
       <!-- TRACK ANALYSIS FORMAT -->
       <item>
        <widget class="QGroupBox" name="groupTrackFormat">
         <property name="title"><string>Track analysis format</string></property>
         <property name="styleSheet"><string>QGroupBox { font-weight: bold; }</string></property>
         <layout class="QGridLayout">
          <property name="verticalSpacing"><number>2</number></property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkIsoMFM">
            <property name="text"><string>ISO MFM</string></property>
            <property name="checked"><bool>true</bool></property>
            <property name="styleSheet"><string>color: #00ff00;</string></property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkEEmu">
            <property name="text"><string>E-Emu</string></property>
            <property name="styleSheet"><string>color: #ffff00;</string></property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkAED6200P">
            <property name="text"><string>AED 6200P</string></property>
            <property name="styleSheet"><string>color: #ff00ff;</string></property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkIsoFM">
            <property name="text"><string>ISO FM</string></property>
            <property name="styleSheet"><string>color: #00ffff;</string></property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkTYCOM">
            <property name="text"><string>TYCOM</string></property>
            <property name="styleSheet"><string>color: #ff8800;</string></property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="checkMEMBRAIN">
            <property name="text"><string>MEMBRAIN</string></property>
            <property name="styleSheet"><string>color: #8888ff;</string></property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkAmigaMFM">
            <property name="text"><string>AMIGA MFM</string></property>
            <property name="checked"><bool>true</bool></property>
            <property name="styleSheet"><string>color: #ff0000;</string></property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkAppleII">
            <property name="text"><string>Apple II</string></property>
            <property name="styleSheet"><string>color: #888888;</string></property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="checkArburg">
            <property name="text"><string>Arburg</string></property>
            <property name="styleSheet"><string>color: #88ff88;</string></property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkC64GCR">
            <property name="text"><string>C64 GCR</string></property>
            <property name="checked"><bool>true</bool></property>
            <property name="styleSheet"><string>color: #4488ff;</string></property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkAtariFM">
            <property name="text"><string>Atari FM</string></property>
            <property name="styleSheet"><string>color: #ff4488;</string></property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="checkUnknown">
            <property name="text"><string>Unknown</string></property>
            <property name="styleSheet"><string>color: #666666;</string></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       
       <!-- TRACK/SIDE SELECTION -->
       <item>
        <widget class="QGroupBox" name="groupSelection">
         <property name="title"><string>Track / Side selection</string></property>
         <property name="styleSheet"><string>QGroupBox { font-weight: bold; }</string></property>
         <layout class="QGridLayout">
          <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Track:</string></property></widget></item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinTrackNumber">
            <property name="minimum"><number>0</number></property>
            <property name="maximum"><number>85</number></property>
            <property name="value"><number>0</number></property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSlider" name="sliderTrack">
            <property name="orientation"><enum>Qt::Horizontal</enum></property>
            <property name="maximum"><number>85</number></property>
           </widget>
          </item>
          <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Side:</string></property></widget></item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinSideNumber">
            <property name="minimum"><number>0</number></property>
            <property name="maximum"><number>1</number></property>
            <property name="value"><number>0</number></property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSlider" name="sliderSide">
            <property name="orientation"><enum>Qt::Horizontal</enum></property>
            <property name="maximum"><number>1</number></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       
       <!-- BUTTONS -->
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="btnEditTools">
           <property name="text"><string>Edit tools</string></property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnExport">
           <property name="text"><string>Export...</string></property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="text"><string>OK</string></property>
          </widget>
         </item>
        </layout>
       </item>
       
      </layout>
     </item>
     
    </layout>
   </item>
   
   <!-- BOTTOM STATUS BAR -->
   <item>
    <widget class="QFrame" name="frameStatusBar">
     <property name="maximumHeight"><number>25</number></property>
     <property name="frameShape"><enum>QFrame::StyledPanel</enum></property>
     <layout class="QHBoxLayout">
      <property name="leftMargin"><number>4</number></property>
      <property name="topMargin"><number>2</number></property>
      <property name="rightMargin"><number>4</number></property>
      <property name="bottomMargin"><number>2</number></property>
      <item>
       <widget class="QLabel" name="labelLibVersion">
        <property name="text"><string>libhxcfe v2.8.14.3</string></property>
        <property name="styleSheet"><string>font-family: monospace;</string></property>
       </widget>
      </item>
      <item><spacer><property name="orientation"><enum>Qt::Horizontal</enum></property></spacer></item>
      <item>
       <widget class="QLabel" name="labelCRC">
        <property name="text"><string>CRC32: 0x00000000</string></property>
        <property name="styleSheet"><string>font-family: monospace;</string></property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
